app/views/approval_notifier_mailer/approval_email.html.erb

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta content='text/html; charset=UTF-8' http-equiv='Content-Type'/>
+</head>
+<body>
+<h2>User <%= @user.full_name %> is requesting access.</h2>
+<p>Please allow or deny.</p>
+<p class=<%= @user.access_token_valid? ? 'active' : 'inactive' %>>
+  <strong>Name:</strong>
+  <%= @user.full_name %>
+</p>
+
+<p>
+  <strong>Phone number:</strong>
+  <a href="tel:<%= @user.phone_number %>"><%= @user.phone_number %></a>
+</p>
+
+<p>
+  <strong>Facility:</strong>
+  <%= @user.facility.name %>
+</p>
+
+<p>
+  <strong>First login in at:</strong>
+  <%= @user.logged_in_at.try(:iso8601) || 'Not logged in yet' %>
```

Comment:
The user would have never logged in on the server when the email is sent. Is this information necessary?